datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -34,19 +45,22 @@ be disconnected.
 Additionally, if UCP is not configured for high availability, you will be
 temporarily unable to:
 
-* Log in to the UCP Web UI
-* Perform CLI operations using existing client bundles
+* Log in to the UCP Web UI.
+* Perform CLI operations using existing client bundles.
 
 To minimize the impact of the backup policy on your business, you should:
 
 * Configure UCP for high availability. This allows load-balancing user requests
 across multiple UCP manager nodes.
 * Schedule the backup to take place outside business hours.
 
-## Backup command
+## Backup the swarm
```

Comment:
I wouldn't make this a whole section, it's just a prerequisite. You can put it at the top of the next section, even as the first step.